__tests__/computed.test.js

```diff
@@ -33,11 +33,61 @@ test('return computed property', async () => {
         me {
           firstName
           lastName
-          fullName @computed(value: "$firstName _&_&_ $lastName")
+          fullName @computed(value: "$me.firstName _&_&_ $me.lastName")
```

Comment:
What's `_&_&_`?

Reply:
Static text. I checking if my regexp work correctly and do not replace this text